contracts/starknet/space/space.cairo

```diff
@@ -119,14 +197,12 @@ func vote{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr : fe
         assert 1 = 0
```

Comment:
can use `with_attr error_message("message"):` for assertions, with a specific failure error message